config/jail.conf

```diff
@@ -820,6 +820,9 @@ udpport = 1200,27000,27001,27002,27003,27004,27005,27006,27007,27008,27009,27010
 action  = %(banaction)s[name=%(__name__)s-tcp, port="%(tcpport)s", protocol="tcp", chain="%(chain)s", actname=%(banaction)s-tcp]
            %(banaction)s[name=%(__name__)s-udp, port="%(udpport)s", protocol="udp", chain="%(chain)s", actname=%(banaction)s-udp]
 
+[centreon]
+logpath = /var/log/centreon/login.log
```

Comment:
Does not the jail expected more here? At least default `port` would be nice.

Reply:
You're right, I've added concerned ports.
Regarding ban times, I've left default values, as done with `apache-auth` for example.
Thank you 👍